src/PostProcess/babylon.postProcessManager.ts

```diff
@@ -147,8 +147,10 @@
             var engine = this._scene.getEngine();
 
             for (var index = 0, len = postProcesses.length; index < len; index++) {
+                var pp = postProcesses[index];
+
                 if (index < len - 1) {
-                    postProcesses[index + 1].activate(camera, targetTexture);
+                    pp._outputTexture = postProcesses[index + 1].activate(camera, targetTexture);

```

Comment:
Only one comment here: making sure Internal texture.dispose will scan all scenes and all pp to remove link to it

Reply:
ok, added to that logic to src/Engine/babylon.engine.ts release method. Let me know if that is how you were thinking.